scripts/check-api-compatibility.sh

```diff
@@ -55,15 +55,22 @@ if ! ${SKIP_DOWNLOAD:-false}; then
     existing_names=$(echo "$jsii_package_dirs" | xargs -n1 -P4 -I {} bash -c 'dirs_to_existing_names "$@"' _ {})
     echo " Done." >&2
 
-    current_version=$(node -p 'require("./lerna.json").version')
-    echo "Current version in lerna.json is $current_version"
-    if ! ${DOWNLOAD_LATEST:-false} && package_exists_on_npm aws-cdk $current_version; then
-        echo "Using package version ${current_version} as baseline"
-        existing_names=$(echo "$existing_names" | sed -e "s/$/@$current_version/")
-    else
-        echo "However, using the latest version from NPM as the baseline"
+    version=$(node -p 'require("./scripts/resolve-version.js").version')
```

Comment:
Should this use `$repo_root` (or `dirname`) to build the path, so you're not at the mercy of whatever working directory this script is executed from?